api/tests/src/db/models/factories.py

```diff
@@ -29,9 +30,121 @@
     OpportunityStatus,
 )
 
-_db_session: Optional[db.Session] = None
+
+class CustomProvider(BaseProvider):
+    """
+    This class is a custom faker provider that can be used to generate
+    fake data for our specific scenarios.
+
+    The name of the functions defined in this class is the name of the individual provider.
+    For example, the "agency" method below can be called by doing either of the following::
+
+        fake.agency()
+
+        factory.Faker("agency")
+
+    Below we register this provider class with both the faker instance we setup, as well as
+    the underlying one backing the factory's faker instance.
+
+    See: https://faker.readthedocs.io/en/master/#how-to-create-a-provider
+    """
+
+    # Ideally use agencies here that actually exist so anyone
+    # using data generated by this for front-end dev actually
+    # has valid values
+    AGENCIES = ["USAID", "ARPAH", "DOC", "DOC-EDA"]
```

Comment:
@rylew1 / @andycochran - FYI updating the factories so that we don't generate the `US-ABC` agencies anymore. Should align with actual values

Reply:
Thanks - looks like this matches some of the choices we already have on the agency filter. 🚀

Reply:
Longer term nits since already approved this pr:
- Would you eventually pull all the actual values or define a type or enum?

Reply:
I'm not sure how we'll manage Agency long-term in the API. But when the front-end has more agencies in its list, we can add a bunch more here uneventfully regardless of approach.